src/AzureTopicConfigurationStrucuture.json

```diff
@@ -0,0 +1,11 @@
+[
+  {
+    "topicName": "apprenticeship_update_created",
+    "subscription": [ "Task_ApprenticeUpdateCreatedMessageProcessor" ]
+  },
+  {
+    "topicName": "cohort_approval_requested_by_provider",
+    "subscription": [ "Task_CohortApprovedMessageProcessor" ]
+  }
+]
+
```

Comment:
The changes is fine but can you remind me why we can't create these subscriptions automatically in code?  It'd be easier than maintaining these json files.

Reply:
This is a MVP version. We needed release this quickly and wanted to take time on coming up with a better solution at a later date.